spec/services/catalog/duplicate_image_spec.rb

```diff
@@ -19,7 +19,7 @@
 
   describe "#process" do
     let!(:base_image) { Image.create(:content => Base64.encode64(File.read(Rails.root.join("spec", "support", "images", "ocp_logo.#{extension}")))) }
-    let(:new_image) { Image.create(:content => Base64.encode64(File.read(Rails.root.join("spec", "support", "images", filename)))) }
+    let(:new_image) { Image.new(:content => Base64.encode64(File.read(Rails.root.join("spec", "support", "images", filename)))) }
```

Comment:
When running with a specific seed this test would fail on first run. This fixes it since it doesn't persist it to the database unless it's actually unique.

Reply:
Moved to #449